script.js

```diff
@@ -1,11 +1,23 @@
 // Create an array called `colors` that contains three colors.
+// var colors = ["red", "yellow", "green"]
 
 // Log the last item in the array to the console.
+// console.log(colors.slice(2))
```

Comment:
:+1: This technically does log the last item in the array, but you only needed to do something like `colors[2]`